### ISSUE TITLE
Slack: A message property returning if it's an edit

### ISSUE THREAD

**Issue by rcoup** [errbotio/errbot#1143]
In order to let us help you better, please fill out the following fields as best you can:

### I am...

* [X] Suggesting a new feature

### I am running...

* Errbot version: 5.1.3
* OS version: Debian (python Docker)
* Python version: 3.6
* Using a virtual environment: no

### Issue description

After #382, when messages are edited in Slack, Errbot plugins receive them again. If the bot responds to a message (eg. "ticket #1234" -> {details of ticket}) then the channel keeps getting additional replies over & over.

Would be nice to be able to detect a message as an edit so the botcmd can skip it. eg. `if msg.is_edit: return` 

This doesn't solve all use cases (eg. "ticket #1234" edited to "ticket #1235" when the bot _should_ respond with different content) but does help the common case, and could be built on to support that sort of thing.

**Comment by andrewthetechie** [comment]
https://github.com/errbotio/errbot/issues/1032 is related to this

**Comment by sijis** [comment]
They are different. This is asking for a property to be added to the `msg` object. The other ticket is looking for a mechanism to have a plugin reply update an existing messge (if the backend supports the feature).

**Comment by andrewthetechie** [comment]
Yup, but they're a bit related imho. If a plugin can update/edit a message, it would be good to be able to check if a new message coming in is edited.

**Comment by sijis** [comment]
Possibly related: https://github.com/errbotio/errbot/pull/1269
